src/components/Drawer/DrawerSection.js

```diff
@@ -70,7 +72,7 @@ class DrawerSection extends React.Component<Props> {
     const fontFamily = fonts.medium;
 
     return (
-      <View {...rest}>
+      <View style={[style, { marginBottom: 4 }]} {...rest}>
```

Comment:
User passed `style` should be at the end
```suggestion
      <View style={[{ marginBottom: 4 }, style]} {...rest}>
```

Reply:
Good catch Satya :D